packages/core/src/BaseSwitch/BaseSwitch.tsx

```diff
@@ -81,6 +81,10 @@ export interface HvBaseSwitchProps
    * @ignore
    */
   onBlur?: (event: React.FocusEvent<any>) => void;
+  /**
+   * The size of the switch.
+   */
+  size?: "sm" | "lg";
```

Comment:
this is actually a breaking change - let's keep the MUI's naming (`small` & `medium`), and leverage what it offers if possible